text-ui-test/EXPECTED.TXT

```diff
@@ -40,16 +40,16 @@ Additionally, you've also added these additional notes!
 Refer to lecture notes.
 Looks like another mission to complete! Let's do it!
 
-2020-10-13:
+2020-10-17:
```

Comment:
Will we have to change the dates EXPECTED.TXT everytime we test?

Reply:
I removed the runtest.sh from GitHub actions in 732c687 because I realized this is not practical.